docs/source/en/api/loaders/ip_adapter.md

```diff
@@ -12,11 +12,11 @@ specific language governing permissions and limitations under the License.
 
 # IP-Adapter
 
-[IP-Adapter](https://hf.co/papers/2308.06721) is a lightweight adapter that enables prompting a diffusion model with an image. This method decouples the cross-attention layers of the image and text features. The image features are generated from an image encoder. Files generated from IP-Adapter are only ~100MBs.
+[IP-Adapter](https://hf.co/papers/2308.06721) is a lightweight adapter that enables prompting a diffusion model with an image. This method decouples the cross-attention layers of the image and text features. The image features are generated from an image encoder.
```

Comment:
Does it make sense to indicate the file size to give an upfront idea about the tinyness of the adapter?

Reply:
I think it's probably ok to not mention file size which seems like an extra detail for a reference page. It's already mentioned in the loading guide (where it is more relevant) which is linked on this page.